### PR TITLE
Test migration of community page to druid.apache.org

### PR DESCRIPTION
**Pull request by jon-wei**
Migrates the community page to druid.apache.org as part of migration SEO testing.

https://github.com/apache/incubator-druid-website/pull/3 should be merged first and confirmed to be up before merging this.

### CODE REVIEW

File: _includes/page_footer.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <a href="/use-cases">Use Cases</a>&ensp;·&ensp;
     <a href="/druid-powered">Powered by Druid</a>&ensp;·&ensp;
     <a href="/docs/latest">Docs</a>&ensp;·&ensp;
-    <a href="/community/">Community</a>&ensp;·&ensp;
+    <a href="https://druid.apache.org/community">Community</a>&ensp;·&ensp;
```
Comment (gianm) [inline _includes/page_footer.html]: This should be https://druid.apache.org/community/ (/community is a redirect so it cannot be canonical).

File: community/index.md [hunk 2]
```diff
@@ -1,158 +1,4 @@
 ---
-title: Apache Druid (incubating) Community
-sectionid: community
-layout: simple_page
+layout: redirect_page
+redirect_target: 'https://druid.apache.org/community'
```
Comment (gianm) [inline community/index.md]: https://druid.apache.org/community/, same comment.

File: _includes/page_header.html [hunk 3]
```diff
@@ -12,7 +12,7 @@
         <li class="{% if page.sectionid == 'use-cases' %} active{% endif %}"><a href="/use-cases">Use Cases</a></li>
         <li class="{% if page.sectionid == 'powered-by' %} active{% endif %}"><a href="/druid-powered">Powered By</a></li>
         <li class="{% if page.sectionid == 'docs' %} active{% endif %}"><a href="/docs/latest/design/">Docs</a></li>
-        <li class="{% if page.sectionid == 'community' %} active{% endif %}"><a href="/community/">Community</a></li>
+        <li class="{% if page.sectionid == 'community' %} active{% endif %}"><a href="https://druid.apache.org/community">Community</a></li>
```
Comment (gianm) [inline _includes/page_header.html]: https://druid.apache.org/community/, same comment.

File: community/index.md [hunk 4]
```diff
@@ -1,158 +1,4 @@
...
 ---
-
-## Community
```
Comment (gianm) [inline community/index.md]: A Markdown doc is leaving here and generated HTML is entering the ASF website repo. I think that's fine for this test, but before migrating more pages we should figure out how to get a Markdown build working in the ASF repo.

File: _includes/page_footer.html (hunk 1 above)
Comment (jon-wei) [reply _includes/page_footer.html]: Fixed

File: community/index.md (hunk 2 above)
Comment (jon-wei) [reply community/index.md]: Fixed

File: _includes/page_header.html (hunk 3 above)
Comment (jon-wei) [reply _includes/page_header.html]: Fixed

File: community/index.md (hunk 4 above)
Comment (jon-wei) [reply community/index.md]: Agreed

**Pull request merged**